test.js

```diff
@@ -62,9 +62,9 @@ const fixtures = [
   {expr: '1 >= 2',          expected: false },
 
   // logical expression lazy evaluation
-  {expr: 'true || false',  expected: true  },
+  {expr: 'true || throw()',  expected: true  },
```

Comment:
```suggestion
  {expr: 'true || false',  expected: true  },
  {expr: 'true || throw()',  expected: true  },
```

Reply:
This throw() is used to test the short-circuit logic and will not be called.
see the same in [donmccurdy/expression-eval](https://github.com/donmccurdy/expression-eval/blob/3ee41060d357887d01beb65a6f3ff2c5a27143f6/test.js#L66)

Reply:
I see, but you still need to keep the original test cases.

Reply:
Could you please tell me why

Reply:
Just to cover all cases.